index.html

```diff
@@ -15,7 +15,7 @@
     <style>
       #root {
         background-color: var(--a-gray-200);
-        min-height: 80vh;
+        min-height: 55vh;
```

Comment:
Jeg husker at jeg økte den for ikke så lenge siden fra 65vh til 80vh, uten at jeg kan huske hvorfor.
Ved å kjøre applikasjonen nå ser jeg ikke noe uheldig konskvens. Foreslår å ta endringen inn og holde øyne åpne på eventuelle layout avvik.

Reply:
Prøvde å sjekke ut med forskjellige skjermstørrelser, men kunne det ha vært at man har økt høyden for å skjule mer av dekoratøren sin footer?

Reply:
Det var i forbindelse med saken om desktop visning. Men jeg fant ikke noe som ikke fungerte nå.